### PR TITLE
Multistore changes

### PR DESCRIPTION
**Pull request by eug-L**

### CODE REVIEW

File: admin/controller/module/tawkto.php [hunk 1]
```diff
@@ -25,24 +25,30 @@ public function index(): void
...
-		if (!empty($stores)) {
-			foreach ($stores as $store) {
-				if ($this->config->get('config_url') == $store['url']) {
-					$store_id = intval($store['store_id']);
-				}
-			}
-		}
+		$data['breadcrumbs'] = array();
+		$data['breadcrumbs'][] = array(
+			'text' => $this->language->get('text_home'),
```
Comment (pyhrus) [inline admin/controller/module/tawkto.php:41]: Did not find text_home in language file, is this loaded from elsewhere ?

File: admin/controller/module/tawkto.php [hunk 2]
```diff
@@ -25,24 +25,30 @@ public function index(): void
...
 		$data['iframe_url'] = $this->getIframeUrl();
 		$data['hierarchy']  = $this->getStoreHierarchy();
 		$data['url'] = array(
 			'set_widget_url' => $this->url->link('extension/tawkto/module/tawkto.setwidget', '', 'SSL') . '&user_token=' . $this->session->data['user_token'],
 			'remove_widget_url' => $this->url->link('extension/tawkto/module/tawkto.removewidget', '', 'SSL') . '&user_token=' . $this->session->data['user_token'],
 		);
-		$data['widget_config']  = $this->getCurrentSettingsFor($store_id);
+
 		$data['same_user'] = true;
 		if (isset($data['widget_config']['user_id'])) {
```
Comment (pyhrus) [inline admin/controller/module/tawkto.php]: loading of widget_config is removed, but it's still needed for checking $data['widget_config']['user_id']

Comment (pyhrus) [reply admin/controller/module/tawkto.php]: also user_id is not loaded in getCurrentSettingsFor

File: admin/view/template/module/tawkto.twig [hunk 3]
```diff
@@ -79,28 +50,51 @@
...
+			}
+
+			if (store.current.widgetId) {
+				src = src + '&currentWidgetId=' + store.current.widgetId;
+			}
+			if (store.current.pageId) {
+				src = src + '&currentPageId=' + store.current.pageId;
+			}
+		}
+		src = src + '&pltf=opencart&pltfv=3&parentDomain=' + currentHost;
```
Comment (pyhrus) [inline admin/view/template/module/tawkto.twig]: pltfv=4

File: admin/controller/module/tawkto.php (hunk 1 above)
Comment (eug-L) [reply admin/controller/module/tawkto.php:41]: added comment

File: admin/controller/module/tawkto.php (hunk 2 above)
Comment (eug-L) [reply admin/controller/module/tawkto.php]: moved user_id checking to js

**Review by pyhrus — APPROVED** [review]

**Pull request merged**